src/components/StripCard/docs/index.story.tsx

```diff
@@ -0,0 +1,95 @@
+import * as React from 'react';
+import { StripCard } from '../';
+import * as examples from './examples';
+import {
+  api,
+  divider,
+  importExample,
+  playground,
+  tab,
+  code as baseCode,
+  tabs,
+  testkit,
+  title,
+} from 'wix-storybook-utils/Sections';
+import { allComponents } from '../../../../stories/utils/allComponents';
+import { Text, TYPOGRAPHY } from '../../Text';
+
+const code = config =>
+  baseCode({ components: allComponents, compact: true, ...config });
+
+const mediaExamples = [
+  {
+    label: 'image',
+    value: (
+      <div
+        style={{
+          background:
+            'url("https://static.wixstatic.com/media/dd7e03deffdd4267b75ef0a60d8510c7.jpg/v1/fill/w_100,h_100,al_c,q_80,usm_0.66_1.00_0.01/Gift%20Wrapped.jpg") no-repeat center center',
+          height: '100%',
+          backgroundSize: 'cover',
+        }}
+      />
+    ),
+  },
+  { label: 'video', value: <video src="" /> },
```

Comment:
How about using a real video here?
https://interactive-examples.mdn.mozilla.net/media/examples/flower.webm for instance?